openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/StreamProcessor.java

```diff
@@ -508,6 +510,12 @@ void addStream(KStream stream) {
 		return streamByUid.values();
 	}
 
+	Collection<KStream> getStreamsByRoom(Long roomId) {
```

Comment:
Maybe the name can be `getByRoom`
It should be obvious StreamProcessor will return Streams :))